packages/component-library/stories/LineChart.story.js

```diff
@@ -104,117 +68,242 @@ const customLegend = legendData => {
 export default () =>
   storiesOf("Component Lib|Charts/Line Chart", module)
     .addDecorator(withKnobs)
-    .add("Simple usage", () => <LineChart data={sampleSimpleData} />)
-    .add("With some props", () => {
-      const data = object("Data", sampleData);
-      const dataKey = text("dataKey", sampleXKey);
-      const dataValue = text("dataValue", sampleYKey);
-      const dataSeries = text("dataSeries", sampleDataSeries);
-      const subtitle = text("Subtitle", sampleSubtitle);
-      const title = text("Title", sampleTitle);
-      const xLabel = text("xLabel", sampleXLabel);
-      const yLabel = text("yLabel", sampleYLabel);
+    .add(
+      "Standard",
+      () => {
+        const sampleTransportationData = [
+          { x: 2001, y: 217309, series: "weekday" },
```

Comment:
Recommend changing the sample data to `{ year: 2001, ridership: 217309, series: "weekday" }` to show more realistic usage